test/features/step_definitions/sdk_steps.rb

```diff
@@ -10,14 +10,23 @@
   }
 end
 
-Given(/^I initiate the verification process using a webcam with(?: (.+)?)?$/) do |locale|
+Given(/^I initiate the verification process using liveness with(?: (.+)?)?$/) do |locale|
   i18n.load_locale(locale)
   steps %Q{
-    Given I navigate to the SDK using a webcam with "#{locale}"
+    Given I navigate to the SDK using liveness with "#{locale}"
     Then I click on primary_button (SDK)
   }
 end
 
+Given(/^I do( not)? have a camera$/) do |has_no_camera|
+  devices = !has_no_camera ? '{ kind: "video" }' : ''
+  @driver.execute_script('window.navigator.mediaDevices.enumerateDevices = () => Promise.resolve([' + devices + '])')
```

Comment:
will this work on browsers without promises? If I remember correctly, for browsers without promise support we just assume there is no camera api available. However, that doesn't mean this test code won't break.

Reply:
It won't but we are only using Chrome